src/com/google/javascript/jscomp/DiagnosticGroups.java

```diff
@@ -37,6 +40,9 @@
   static final DiagnosticType UNUSED =
       DiagnosticType.warning("JSC_UNUSED", "{0}");
 
+  public static final Set<String> wildcardExcludedGroups =
+      new HashSet<String>(Arrays.asList(new String[] { "reportUnknownTypes" }));
```

Comment:
:+1: 
There might be other things that belong in this list but none come to mind immediately.